kayvee.go

```diff
@@ -27,6 +28,9 @@ const (
 
 // Format converts a map to a string of space-delimited key=val pairs
 func Format(data map[string]interface{}) string {
+	if os.Getenv("_DEPLOY_ENV") != "" {
```

Comment:
The env should always override the input data, but we can cache the env lookup: https://github.com/Clever/kayvee-go/pull/23/commits/3a004af053ef2a4e981f5bef6c146859f9d1d8dc